src/webapi/date.rs

```diff
@@ -1,25 +1,1166 @@
+    #[inline]
+    fn set_seconds( &self, seconds: i32, _ms: Option< i32 > ) {
+        match _ms {
+            Some(_ms) => {
+                js! { @(no_return)
+                    @{self.as_ref()}.setSeconds(@{seconds}, @{_ms});
+                };
+            },
+            None => {
+                js! { @(no_return)
+                    @{self.as_ref()}.setSeconds(@{seconds});
+                };
+            }
+        }
+    }
+
+    /// The set_time() method sets the Date object to the time represented by a number of
+    /// milliseconds since January 1, 1970, 00:00:00 UTC.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setTime)
+    #[inline]
+    fn set_time( &self, time_value: f64 ) {
+        js! { @(no_return)
+            @{self.as_ref()}.setTime(@{time_value});
+        };
+    }
+
+    /// The set_utc_date() method sets the day of the month for a specified date according to
+    /// universal time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCDate)
+    #[inline]
+    fn set_utc_date( &self, day_value: i32 ) {
+        js! { @(no_return)
+            @{self.as_ref()}.setUTCDate(@{day_value});
+        };
+    }
+
+    /// The set_utc_full_year() method sets the full year for a specified date according to universal
+    /// time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCFullYear)
+    #[inline]
+    fn set_utc_full_year( &self, year: i32, _month: Option< i32 >, _day: Option< i32 > ) {
+        match _month {
+            Some(_month) => match _day {
+                Some(_day) => {
+                    let args_obj: HashMap< &str, Value > = [
+                        ("year",  Value::Number( year.into())),
+                        ("month", Value::Number( _month.into())),
+                        ("day",   Value::Number( _day.into()))
+                    ].iter().cloned().collect();
+
+                    js! { @(no_return)
+                        var args = @{args_obj};
+                        @{self.as_ref()}.setUTCFullYear(args.year, args.month, args.day);
+                    };
+                },
+                None => {
+                    js! { @(no_return)
+                        @{self.as_ref()}.setUTCFullYear(@{year}, @{_month});
+                    };
+                }
+            },
+            None => {
+                js! { @(no_return)
+                    @{self.as_ref()}.setUTCFullYear(@{year});
+                };
+            }
+        }
+    }
+
+    /// The set_utc_hours() method sets the hour for a specified date according to universal time,
+    /// and returns the number of milliseconds since 1 January 1970 00:00:00 UTC until the time
+    /// represented by the updated Date instance.
+    ///
```

Comment:
Please get rid of the method name from all of these doc comments, e.g.:

`The set_utc_hours() method sets the hour` -> `Sets the hour`